fem/complex_fem.cpp

```diff
@@ -326,37 +361,105 @@ SesquilinearForm::FormLinearSystem(const Array<int> &ess_tdof_list,
    Vector b_i(&(b.GetData())[vsize], vsize);
    b_i *= s;
```

Comment:
Instead of scaling by s, which could be 1, it would be more efficient to avoid scaling by 1 by using an if-statement. Instead of storing s, just store a bool and scale by -1 if necessary.

Reply:
done!